views/layout.pug

```diff
@@ -0,0 +1,24 @@
+doctype html
+html(lang='en')
+
+  head
+    meta(charset='utf-8')
+    meta(http-equiv='X-UA-Compatible' content='IE=edge')
+    meta(name='viewport' content='width=device-width, initial-scale=1')
+
+    link(rel='stylesheet' href='bootstrap/css/bootstrap.min.css')
+    link(rel='stylesheet' href='materializecss/css/materialize.min.css')
+    link(rel='stylesheet' href='/css/icons.css')
+
+    script(src='jquery/jquery.js')
```

Comment:
not sure if scripts should go after the body. If the scripts load and try to grep elements from the body, but the body hasnt loaded yet, would the scripts fail?

Reply:
please explain this further

Reply:
I was wondering since we are loading the scripts before we load the body, would that cause issues if the scripts try to do something like $('some element').val() in the $() function but the element has not been loaded yet since the body has not been complied by pug. Am not sure exactly if this is an issue or not, just wondering if you know

Reply:
I dont really think that this is an issue

Reply:
great, thanks for confirming